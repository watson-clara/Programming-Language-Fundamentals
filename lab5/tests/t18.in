son('Prince Albert', Y),
print(Y),
print('\n'),
halt.
