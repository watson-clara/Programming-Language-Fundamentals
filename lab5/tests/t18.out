Ernest I: Duke of Saxe-Coburg and Gotha
